### PR TITLE
New requirements for the init cmd

### PR DESCRIPTION
**Pull request by kosegor**
## Scope
[MP-868](https://madhive.atlassian.net/browse/MP-868)

## Why?
As an alicenet user, I would like to facilitate and guide the init execution for a node runner as much as I can.

### CODE REVIEW

**Comment by vtleonardo** [comment]
Bug: trim whitespaces on user input before saving to the toml.
Add: unit tests for uncovered part:
layer1/ethereum/ethereum.go#L225-L228

**Review by vtleonardo — CHANGES_REQUESTED** [review]
Bug: trim whitespaces on user input before saving to the toml.
Add unit tests for uncovered part:
layer1/ethereum/ethereum.go#L225-L228

**Comment by kosegor** [comment]
closes #416

File: cmd/ethkey/utils.go [hunk 1]
```diff
@@ -155,5 +155,7 @@ func ReadInput(message string) (string, error) {
 		return "", err
 	}
 
+	input = strings.TrimSpace(input)
```
Comment (vtleonardo) [inline cmd/ethkey/utils.go:158]: @kosegor does this also remove trailing characters? What happened is that, when I typed on terminal an Ethereum URL and pressed enter, the " (quotation symbol) got placed in a new line in the toml

Comment (kosegor) [reply cmd/ethkey/utils.go:158]: yes, I noticed that and it's also fixed

**Review by vtleonardo — APPROVED** [review]

**Pull request merged**